phases/51-gnustep-back.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -eo pipefail
+shopt -s inherit_errexit
+
+cd `dirname $0`
+
+export PROJECT=gnustep-back
+export GITHUB_REPO=gnustep/libs-back
+export TAG=
+
+# load environment and prepare project
+../scripts/common.bat prepare_project
+
+cd "$SRCROOT/$PROJECT"
+
+echo
+echo "### Loading GNUstep environment"
+. "$UNIX_INSTALL_PREFIX/share/GNUstep/Makefiles/GNUstep.sh"
+
+echo
+echo "### Running configure"
+
+./configure \
+    --enable-graphics=headless \
+	--enable-server=headless \
```

Comment:
Nit: do you mind adjusting the indentation here to match the rest (I think 4 spaces)?

Reply:
Ah, missed that one.  Thanks for spotting it!  Should be fixed now.